src/Microsoft.Health.Dicom.Functions.App/host.json

```diff
@@ -52,6 +52,16 @@
         "MaxNumberOfAttempts": 4
       }
     },
+    "MigratingFiles": {
+      "BatchSize": 100,
+      "MaxParallelThreads": 5,
```

Comment:
Remove `BatchSize` and `MaxParallelBatches`

Reply:
BatchSize and MaxParallelBatches are used to get instance batches similar to re-indexing. I could make it as constant. Is that what you are suggesting?

Reply:
Removed